composer/composer_transform_csv_to_json.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable func-style */
```

Comment:
any reason why we don't use arrow functions here?

Reply:
This function is basically a more robust version of this function published here: 
https://cloud.google.com/dataflow/docs/guides/templates/provided-streaming#gcstexttobigquerystream

I'm not entirely sure, but I think BQ might get mad if we switch the style on it. I may be wrong, just didn't want to deviate too much from the sample for consistency!